corehq/apps/cloudcare/templates/form_entry/templates.html

```diff
@@ -201,7 +201,8 @@ <h1 class="title" data-bind="text: title, visible: !showInFormNavigation()"></h1
 
 <script type="text/html" id="question-fullform-ko-template">
   <div data-bind="class:questionTileWidth">
-  <!-- ko if: datatype() !== 'info' -->
+  <!-- ko if: datatype() !== 'info'
+    && !(datatype() === 'select' && stylesContains('button-select')) -->
```

Comment:
This conditional is really ugly. I considered adding it [here](https://github.com/dimagi/commcare-hq/blob/master/corehq/apps/cloudcare/static/cloudcare/js/form_entry/form_ui.js#L730) as a property on the question, but after seeing that done with `self.isLabel` and that _not_ being used as the knockout conditional here, I sort of assumed there was a good reason (avoiding any semblance of circular dependencies, maybe?)